bot/exts/fun/quack.py

```diff
@@ -0,0 +1,75 @@
+import random
+from typing import Optional
+
+import discord
+from aiohttp import client_exceptions
+from discord.ext import commands
+
+from bot.bot import Bot
+from bot.constants import Colours, NEGATIVE_REPLIES
+
+API_URL = 'https://quackstack.pythondiscord.com'
+
+
+class Quackstack(commands.Cog):
+    """Cog used for wrapping Quackstack."""
+
+    def __init__(self, bot: Bot) -> None:
```

Comment:
We no longer have this annotation across this repo for the dunder of `__init__`.

```suggestion
    def __init__(self, bot: Bot):
```